functions/__sf_section_julia.fish

```diff
@@ -0,0 +1,37 @@
+#
+# Julia
+#
+# Current Julia version.
+
+function __sf_section_julia -d "Display julia version"
+    # ------------------------------------------------------------------------------
+    # Configuration
+    # ------------------------------------------------------------------------------
+
+    __sf_util_set_default SPACEFISH_JULIA_SHOW true
+    __sf_util_set_default SPACEFISH_JULIA_PREFIX "is "
+    __sf_util_set_default SPACEFISH_JULIA_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+    __sf_util_set_default SPACEFISH_JULIA_SYMBOL "ஃ "
+    __sf_util_set_default SPACEFISH_JULIA_COLOR green
+
+    # ------------------------------------------------------------------------------
+    # Section
+    # ------------------------------------------------------------------------------
+
+    [ $SPACEFISH_JULIA_SHOW = false ]; and return
+
+    # Show Julia version only if julia is installed
+    type -q julia; or return
+
+    # Show julia version only when pwd has *.jl file(s)
```

Comment:
Just [for consistency](https://github.com/matchai/spacefish/blob/7134fd46310615ce97410107b370b96fa99ff96c/functions/__sf_section_golang.fish#L31), I would suggest testing with the following manner for *.jl files:
[ (count *.jl) -gt 0 ]; or return